doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2002,70 +2002,70 @@ The fully expanded example above (without environment variables) looks like this
   curl -H "X-Dataverse-key:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" -X DELETE "https://demo.dataverse.org/api/datasets/2347/assignments/6"
 
 
-Create a Private URL for a Dataset
+Create a Preview URL for a Dataset
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Should we add a note somewhere that the API might return with mention of "Private URL" and that this is expected, that it's simply the old name?

Reply:
added a note for create